### ISSUE TITLE
ListBox: Incorrect value for "aria-multiselectable" attribute ( WCAG: 4.1.2 violation)

### ISSUE THREAD

**Issue by lcameron-vs** [primefaces/primeng#11743]
### Describe the bug

Template for Listbox in `src/app/components/listbox/listbox.ts` incorrectly assigns the `aria-multiselectable` attribute a value of the string `"multiple"`. This results in an invalid value for the attribute; should only be `true | false`.

`ListBox` has a `boolean` input `multiple`, so this binding should change:
```
-            <ul class="p-listbox-list" role="listbox" aria-multiselectable="multiple">
+            <ul class="p-listbox-list" role="listbox" [attr.aria-multiselectable]="multiple">
```

### Environment

N/A

### Angular version

N/A

### PrimeNG version

14.0.0

### Build / Runtime

Angular CLI App

### Language

ALL

### Node version (for AoT issues node --version)

N/A

### Steps to reproduce the behavior

1. Go to http://primefaces.org/primeng/listbox
2. Right-click and inspect HTML for any listbox
3. Inspect `aria-multiselectable` value  for `ul.p-listbox-list`

Actual result:
`<ul class="p-listbox-list" role="listbox" aria-multiselectable="multiple">`

### Expected behavior

Expected result(s):
`<ul class="p-listbox-list" role="listbox" aria-multiselectable="true | false">`

**Comment by cetincakiroglu** [comment]
Hi,

Accessibility is a major concern of the Prime UI libraries and PrimeNG is no exception. [PrimeTek](https://www.primetek.com.tr/) teams have initiated a significant process to review and enhance the accessibility features of the components. This guide documents the foundation of the general guidelines that PrimeNG will follow and each component documentation will have a separate Accessibility section that states the keyboard support, screen reader compatibility, the implementation details along with tips to achieve WCAG compliancy. This work has been initiated in Q2 2022 and planned to be completed by early Q3. PrimeNG will be the reference implementation which then will be ported to PrimeFaces, PrimeNG and PrimeVue.
